canonical-web-and-design/charmhub.io
# Implement the vanilla search bar, pass through search queries

## Done

- Add Vanilla search bar
- When submitting a search, reload the page with the q parameter set and the searched term in the search box

NOTE: This will not filter the results yet as the API does not support this

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Visit http://0.0.0.0:8045/store
- Type something in the search box and press return, or click the search icon
- The page should reload and show the search term in the search box and the query in the url


## Issue / Card

Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1292
